goinstant/cli.go

```diff
@@ -74,7 +61,15 @@ Commands:
 			err = loadIGpackage(startupCommands[1], startupCommands[2], params)
 		}
 	default:
-		fmt.Println("The deploy command is not recognized: ", startupCommands)
+		if len(startupCommands) < 2 {
+			fmt.Println("The deploy command is not recognized: ", startupCommands)
```

Comment:
fmt.Println() adds a space by itself. So, the space at 'recognized: ' is not necessary.